# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pinentry/glib-2.16.1.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Conflict with glibc-2.16.1's gmem.h prototypes

--- ./gtk+-2/gtksecentry.c.orig	2009-04-11 13:18:31.000000000 +0300
+++ ./gtk+-2/gtksecentry.c	2009-04-11 13:20:47.000000000 +0300
@@ -270,7 +270,7 @@
 
 
 gpointer
-g_malloc(gulong size)
+g_malloc(gsize size)
 {
     gpointer p;
 
@@ -288,7 +288,7 @@
 }
 
 gpointer
-g_malloc0(gulong size)
+g_malloc0(gsize size)
 {
     gpointer p;
 
@@ -308,7 +308,7 @@
 }
 
 gpointer
-g_realloc(gpointer mem, gulong size)
+g_realloc(gpointer mem, gsize size)
 {
     gpointer p;
 
